.drone.yml

```diff
@@ -41,4 +37,50 @@ pipeline:
       - 'latest'
     when:
       event: push
-      branch: master
\ No newline at end of file
+      branch: master
+
+  e2e_test:
+    image: '${DOCKER_REPO}/${DOCKER_PATH}/${NAME}'
+    secrets: [ service_now_username, service_now_password ]
+    description: 'End-to-end test for the ServiceNow drone plugin. Build #${DRONE_BUILD_NUMBER}'
```

Comment:
I don't understand what this will do. We might need a special account from the ServiceNow team for this.

Reply:
I don't think there's any kind of order in the SN world - I was happily chucking stuff at it the other day to try and work out what failed requests look like, and how to alert plugin users of a problem - nothing came back any other way than `201`, and I was just putting dummy values in for everything.

Reply:
Yeah. The only feedback I'm aware of is failed auth and missing `internal-indentifier`.

Reply:
It's ok, I've done quite a bit to handle the various possible scenarios.